composer.json

```diff
@@ -19,7 +19,7 @@
     ],
     "bin": ["bin/satis"],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.6 || ^7.0",
```

Comment:
It's not a hard requirement. You can probably still get it to run if you do a `composer install|update --ignore-platform-reqs`. However, going forward we will no longer officially support EOL PHP versions for Satis.